templates/availability/service-availability-editor.phtml

```diff
@@ -47,7 +47,7 @@
                     <div class="form-row__input">
                         <div class="form-row__input-with-description">
                             <datetime-picker class="datetime-picker--inline"
-                                             v-validate="'required|date_format:' + config.formats.datetime.tzFree + '|after:$start'"
+                                             v-validate="'required|date_format:' + config.formats.datetime.tzFree + '|after:$start' + (model.isAllDay ? ',true' : '')"

```

Comment:
I have no idea what this means. It looks like some validation rule string, but I have no idea what the optional `true` parameter is.

Reply:
Also, nothing about this PR tells me what the fix is, i.e. what this fixes and how. I cannot make sense of this sentence from the issue, either:

> Validation rule wait for time in format, which results in error because real data have no time.

Reply:
@romalytvynenko, even better - a comment, so that a developer who doesn't know about this PR can still understand the code.